DUI3-DX/Converters/Autocad/Speckle.Converters.AutocadShared/Speckle.Converters.AutocadShared.projitems

```diff
@@ -13,14 +13,29 @@
     <Compile Include="$(MSBuildThisFileDirectory)AutocadConverterToSpeckle.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)AutocadConversionContextStack.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)AutocadToSpeckleUnitConverter.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)DependencyInjection\AutofacAutocadConverterModule.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Geometry\BoxToSpeckleConverter.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)ToHost\Geometry\MeshToHostConverter.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)ToHost\Raw\PolycurveToHostSplineRawConverter.cs" />
+    <Compile Include="..\Speckle.Converters.AutocadShared\ToHost\Geometry\CircleToHostConverter.cs" />
```

Comment:
Did you add this line manually @oguzhankoral. I'm curious why it doesn't use `MSBuildThisFileDirectory` like the others...
Maybe @AlanRynne knows more than I on this

Reply:
I haven't add manually anything on these files 🤔 let me fix it